stage/services/ors/task-definitions/wsgi.json

```diff
@@ -1,49 +1,28 @@
 [
    {
-      "name": "wsgi-stage",
+      "name": "wsgi",
       "image": "jackofsum/ors-wsgi:latest",
       "essential": true,
-      "portMappings" : [],
+      "portMappings" : [
+         {
+            "containerPort": 80,
+            "hostPort": 80
+         }
+      ],
       "environment": [ 
          {"name": "PROXY_URL", "value": "${proxy_url}" },
          {"name": "NEO_PASSWORD", "value":"${neo_password}"},
          {"name": "NEO_USER", "value":"${neo_user}"},
          {"name": "NEO_URL", "value":"${neo_url}"}
       ],
-      "entrypoint": ["uwsgi", "--ini", "new.ini"],
-      "portMappings" : [
-        {
-          "containerPort": 80,
-          "hostPort": 80
-        }
-      ],
-      "logConfiguration": { 
-        "logDriver": "awslogs",
-        "options": { 
-          "awslogs-group" : "/ecs/wgsi-stage",
-          "awslogs-region": "eu-west-1",
-          "awslogs-stream-prefix": "ecs"
-        }
-      }
-   },
-   {
-      "name": "nginx",
-      "image": "jackofsum/ors-nginx",
-      "essential": true,
-      "portMappings" : [
-         {
-            "containerPort": 80,
-            "hostPort": 80
+      "entrypoint": ["uwsgi", "--ini", "http.ini"],
```

Comment:
@mlev71 please use the conventions for logs we use elsewhere. Makes it easier to find the logs. The test infrastructure runs in eu-west-1.

Reply:
Does this work now?

"logConfiguration": { 
         "logDriver": "awslogs",
         "options": { 
            "awslogs-group" : "/ecs/wsgi-stage",
            "awslogs-region": "eu-west-1",
            "awslogs-stream-prefix": "ecs"
            }
      }